src/connection.ts

```diff
@@ -300,7 +300,7 @@ export class StableWSConnection<StreamChatGenerics extends ExtendableGenerics =
       }
     } catch (err) {
```

Comment:
```suggestion
    } catch (err: any) {
```

Instead of casting it below, we can do it here, right?